Source/WebKit/GPUProcess/GPUProcess.cpp

```diff
@@ -236,9 +236,11 @@ void GPUProcess::initializeGPUProcess(GPUProcessCreationParameters&& parameters)
     SandboxExtension::consumePermanently(parameters.microphoneSandboxExtensionHandle);
 #endif
 #if PLATFORM(IOS_FAMILY)
-    CoreAudioSharedUnit::unit().setStatusBarWasTappedCallback([this](auto completionHandler) {
-        parentProcessConnection()->sendWithAsyncReply(Messages::GPUProcessProxy::StatusBarWasTapped(), [] { }, 0);
-        completionHandler();
+    CoreAudioSharedUnit::unit().setStatusBarWasTappedCallback([process = WeakPtr<GPUProcess> { *this }](auto completionHandler) {
+        if (process) {
```

Comment:
We should store this process in RefPtr like so:
if (RefPtr process = weakProcess.get()) {
    process->parentProcessConnection()...
}